.github/workflows/lint.yml

```diff
@@ -21,7 +21,16 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
       - name: install dependencies
-        run: yarn install || echo "ignore failure"
+        run: yarn install
+      - name: Prepare Integration
+        run: ./integration/pbjs.sh
+      # This will fail if any git-tracked file has changed
```

Comment:
It will fail? That would be great if so, but I don't think we already had that implemented, and I'm not seeing any new code here or in `yarn tsc:check` that would detect it?

Fwiw we do this for our internal project, and it looks like it's as simple as adding a `git status && git diff --exit-code`, but I hadn't gotten around to verifying + copy/pasting that over.

Do we need something like that `git diff --exit-code`, or am I missing how this comment is already implemented?

Reply:
Ah I see! Yep, looks like the comment was in slightly the wrong place in `build.yml`; it should have been applied to the next step & not the tsc compile step.

I went ahead and moved the comment in `build.yml` to the more-correct spot + updated the `lint.yml` to do the same `git diff` check for good measure, and just pushed those up as a small chore commit.

Thanks!